exec/csnex/lib/console.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.IO;
+using System.Security.Cryptography;
```

Comment:
What is this using statement for? Surely this module doesn't do any cryptography.

Reply:
Fixed from online edit.